### PR TITLE
Adding plot_finder_image docs, importing it into astroplan.plots namespace

### PR DESCRIPTION
**Pull request by bmorris3**
There was a mistake and an omission in PR #115 -- `plot_finder_image` was not imported into the `astroplan.plots` namespace, and no plotting tutorials were added to the docs (Issue #129). This PR seeks to fix both of those things, while cleaning up the function's docstring.

I've assigned @cdeil -- if I'm being presumptuous you don't have a few minutes to take a look in the next day or so let me know.

cc @tobyrsmith

### CODE REVIEW

**Comment by bmorris3** [comment]
@cdeil Do you know if [this error](https://travis-ci.org/astropy/astroplan/jobs/86455663#L683) in the sphinx build, i.e.:
```
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/matplotlib/sphinxext/plot_directive.py", line 506, in run_code
    six.exec_(code, ns)
File "<string>", line 6, in <module>
File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/astropy/utils/decorators.py", line 522, in plot_finder_image
**_get_function_args(wrapped))
File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/astropy/units/decorators.py", line 130, in wrapper
return wrapped_function(*func_args, **func_kwargs)
File "/home/travis/build/astropy/astroplan/build/lib.linux-x86_64-3.4/astroplan/plots/finder.py", line 88, in plot_finder_image
survey=survey, radius=fov_radius, grid=grid)[0][0]
File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/astroquery/skyview/core.py", line 203, in get_images
return [obj.get_fits() for obj in readable_objects]
File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/astroquery/skyview/core.py", line 203, in <listcomp>
return [obj.get_fits() for obj in readable_objects]
File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/astroquery/utils/commons.py", line 381, in get_fits
filedata = self.get_string()
File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/astroquery/utils/commons.py", line 435, in get_string
data = f.read()
File "/home/travis/miniconda/envs/test/lib/python3.4/codecs.py", line 319, in decode
(result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: \'utf-8\' codec can\'t decode byte 0xa0 in position 14410: invalid start byte
```
is a py2->3 issue within astroquery?

File: docs/tutorials/plots.rst [hunk 1]
```diff
@@ -1057,3 +1059,39 @@ make multiple plots:
...
 
 :ref:`Return to Top <plots>`
+
+.. _finder_image:
+
+Finder Chart/Image
+==================
+
+`astroplan` includes a function for generating quick finder images from the
+command line, `~astroplan.plots.plot_finder_image`, by querying for images from
```
Comment (cdeil) [inline docs/tutorials/plots.rst]: With "from the command line" you mean the interactive Python prompt?
When I read "from the command line" I assume the terminal / shell, not Python, but I think that's not possible, right?
So can you rephrase here to make this more clear?

File: astroplan/plots/finder.py [hunk 2]
```diff
@@ -66,6 +68,12 @@ def plot_finder_image(target, survey='DSS', fov_radius=10*u.arcmin,
 
     hdu : `~astropy.io.fits.PrimaryHDU`
         FITS HDU of the retrieved image
+
+
+    Notes
+    -----
+    Dependencies:
+        In addition to Matplotlib, this function makes use of astroquery.
```
Comment (cdeil) [inline astroplan/plots/finder.py]: I think this also depends on wcsaxes, right?
If yes, you should mention this dependency here also.

**Comment by cdeil** [comment]
@bmorris3 So the UnicodeDecodeError from astroquery happens only on Python 3? Is it the same as https://github.com/astropy/astroquery/issues/593 or a different issue?

File: docs/tutorials/plots.rst [hunk 3]
```diff
@@ -1057,3 +1059,39 @@ make multiple plots:
...
+sky surveys centered on a `~astroplan.FixedTarget`. This function depends on
+`astroquery <https://astroquery.readthedocs.org/en/latest/>`_  (in addition
+to `Matplotlib`_). In this example, we'll quickly make a finder image centered
+on The Crab Nebula (M1):
+
+.. code-block:: python
```
Comment (cdeil) [inline docs/tutorials/plots.rst]: Is it necessary / useful to duplicate this code example?
Can't you add `:include-source:` to the `.. plot::` directive and avoid the code duplication that way?

Comment (bmorris3) [inline docs/tutorials/plots.rst]: I didn't know this command, and it seems not to be working. Should that be placed like so: 
```

.. plot::
   :include-source:
    from astroplan import FixedTarget
    from astroplan.plots import plot_finder_image
    import matplotlib.pyplot as plt

    messier1 = FixedTarget.from_name("M1")
    ax, hdu = plot_finder_image(messier1)
    plt.show()
```

**Comment by bmorris3** [comment]
@cdeil It looks like those errors were just bad builds, they're gone now. I also updated the docstring that you asked to clarify. 

The last remaining error in the travis build is from the sun seperation error that @eteq is working on in #138, so I'm going to mark this PR for final review and merge later unless you do first.

**Comment by bmorris3** [comment]
Merging after no further comments came in.
